src/gradualizer_file_utils.erl

```diff
@@ -58,19 +58,12 @@ epp_parse_file(File, Includes) ->
             Error1
     end.
 
-% epp:open/5 was removed in OTP-24
-% https://github.com/erlang/otp/commit/5281a8c7f77d45a3c36fca9c1a2e4d3812f6fc3d#diff-580a349c49b1d9b5415166e18f5279728d934efe0cebc4ee5a87823055ec3413
 epp_open(File, Fd, StartLocation, Includes) ->
     code:ensure_loaded(epp),
-    case erlang:function_exported(epp, open, 5) of
-        true ->
-            epp:open(File, Fd, StartLocation, Includes, []);
-        false ->
-            epp:open([{name, File},
-                      {location, StartLocation},
-                      {includes, Includes},
-                      {fd, Fd}])
-    end.
+    epp:open([{name, File},
+              {location, StartLocation},
```

Comment:
Apparently, this doesn't give us column numbers in OTP < 24.

Btw, code that depends on OTP version by `erlang:function_exported/3` is an interesting and valid use case. It's the dynamic nature of dynamic loading. If we work around it here, perhaps we can add it to known problems instead?

It can't really be solved, but we can come up with some recommended way to silence it.

Reply:
It can and I solved it in another way, but then decided it's better to remove code than to add more. I'll push the alternative solution instead, since you mention this.

BTW, I don't know why, but dropping this leads to the CI failure on 23.3.4.7 - is that connected with the missing column numbers? The other solution I have in mind passed CI.

Reply:
Yes, I believe it has to do with column numbers. Some OTPs ago, there were no column numbers and then it was added gradually in different modules. (Our Peter Gömöri was involved in adding column numbers in OTP.)

Reply:
I've pushed the alternative approach. It's based on `--specs_override_dir` and providing specs of all the functions we expect to use in various OTP versions. WDYT?